src/pages/about/[mmsi].jsx

```diff
@@ -42,35 +42,38 @@ const Description = () => {
 			</div>
 		);
 	};
-
+	console.log(name);
 	return (
 		<Layout>
 			<Head>
 				<title key="title">{name} || description</title>
 				<meta key="description" name="description" content="" />
 			</Head>
 			<Typography variant="h2" component="h1">
-				<LocationSearchingIcon />
 				{name}
+				<Shipicon />
 			</Typography>
 			<Card>
-				<CardContent>
-					<ul>
-						<li>Latitude : {data && data.latitude}</li>
-						<li>Longitude : {data && data.longitude}</li>
-						<li>Speed : {data && data.speed}</li>
-						<li>TimeStamp: {data && data.timestamp}</li>
-						<li>Status from: {data && unixTime}</li>
-					</ul>
+				<List>
+					<ListItem>Latitude: {data && data.latitude}</ListItem>
+					<ListItem>Longitude: {data && data.longitude}</ListItem>
+					<ListItem>Speed: {data && data.speed}</ListItem>
+					<ListItem>TimeStamp: {data && data.timestamp}</ListItem>
+					<ListItem>Status from: {data && unixTime}</ListItem>
+
 					<Button
 						onClick={() => {
 							reload();
 						}}
 					>
 						Refresh
 					</Button>
-					<Link href={`/missions/${mmsi}`}>Show Missions</Link>
-				</CardContent>
+					<Button>
+						<Link href={`/missions/${mmsi}`} style={{ textDecoration: "none" }}>
+							Show Missions
+						</Link>
+					</Button>

```

Comment:
`Link` should wrap the `Button`
```suggestion
					<Link href={`/missions/${mmsi}`} style={{ textDecoration: "none" }}>
						<Button>
							Show Missions
						</Button>
					</Link>

```